lib/releases.sh

```diff
@@ -43,9 +43,10 @@ else
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.2.")}"
 fi
 
-# Remove the following line when CAPI release v1.3.0 is available
+# Remove the following lines when CAPI/CAPM3 v1.3.0 releases are available
 if [ "${CAPM3RELEASEBRANCH}" == "main" ]; then
  export CAPIRELEASE="v1.3.0-beta.1"
+ export CAPM3RELEASE="v1.3.0-beta.0"
```

Comment:
cannot we force this workaround from capm3 so it can be easier to test full changes when introducing and removing this changes 
here https://github.com/metal3-io/cluster-api-provider-metal3/blob/7fcfe8a3beef3ddeb096366e15a8779936c2be50/scripts/ci-e2e.sh#L23

Reply:
we can, but we want this release to be used not only in capm3 & e2e tests, but with whole CI tests in general.